packages/obonode/obojobo-sections-assessment/adapter.js

```diff
@@ -16,10 +16,29 @@ const Adapter = {
 			model.modelState.rubric = new AssessmentRubric()
 		}
 
-		model.setStateProp(
-			'attempts',
-			Infinity,
-			p => (('' + p).toLowerCase() === 'unlimited' ? Infinity : parseInt(p, 10))
+		if (attrs && attrs.content && attrs.content.lockAssessment) {
```

Comment:
I'm a little weary of adding a new attribute to content for this, since it sorta covers the same use case as the triggers and kind of modifies the document to add them in. I think I'd rather the WYSIWYG editor simply adds or removes these triggers.